service-front/module/Application/src/Model/Service/Lpa/ContinuationSheets.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Application\Model\Service\Lpa;
+
+use Application\Model\Service\AbstractService;
+use Opg\Lpa\DataModel\Lpa\Formatter as LpaFormatter;
+use Opg\Lpa\DataModel\Lpa\Lpa;
+
+class ContinuationSheets
+{
+    /**
+     * Gathers an array on conditions where continuation sheet(s) would be generated in the PDF.
+     *
+     * @param Lpa $lpa
+     * @return array
+     */
+    public function getContinuationNoteKeys(Lpa $lpa): array
+    {
+        //Array of keys to know which extra notes to show in template for continuation sheets
+        $continuationNoteKeys = array();
+        $extraBlockPeople = null;
+        $paCount = count($lpa->document->primaryAttorneys);
+        $raCount = count($lpa->document->replacementAttorneys);
+        $pnCount = count($lpa->document->peopleToNotify);
+
+        if ($paCount > 4 && $raCount > 2 && $pnCount > 4) {
```

Comment:
switch/case might make sense here, as you're doing a lot of comparisons on numbers and it might make it easier to read; not necessary to make this change as this works perfectly well, but worth considering

Reply:
Did a bit of a comparison and agree with you that it is more suited to use a switch statement. The need to break out of the cases makes it longer but it does make it a little easier to read so I've made the change alongside the other :)